# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1330755267 28800
#      Fri Mar 02 22:14:27 2012 -0800
# Node ID 099bd779466c599a3fb545d155d1e6a1b98b4d59
# Parent  d07e989686b0259ed18139c856faf00c7633d42a
test: Fix failing %!test in fileread.m

* fileread.m: Fix failing %!test.

diff --git a/scripts/io/fileread.m b/scripts/io/fileread.m
--- a/scripts/io/fileread.m
+++ b/scripts/io/fileread.m
@@ -48,16 +48,16 @@ endfunction
 
 %!test
 %! cstr = {"Hello World", "The answer is 42", "Goodbye World"};
 %! fname = tmpnam ();
 %! fid = fopen (fname, "w");
 %! fprintf (fid, "%s\n", cstr{:});
 %! fclose (fid);
 %! str = fileread (fname);
-%! assert (str', [cstr{1} "\n" cstr{2} "\n" cstr{3} "\n"]);
+%! assert (str, [cstr{1} "\n" cstr{2} "\n" cstr{3} "\n"]);
 %! unlink (fname);
 
 %% Test input validation
 %!error fileread ()
 %!error fileread (1, 2)
 %!error <FILENAME argument must be a string> fileread (1)
 
